packages/bits/spec/components/checkbox-group/checkbox-group.e2e.ts

```diff
@@ -1,24 +1,27 @@
+import { by, element } from "protractor";
 import { Atom } from "../../atom";
 import { Helpers } from "../../helpers";
 import { CheckboxAtom, CheckboxGroupAtom } from "../public_api";
+import { ButtonAtom } from "../public_api";
+
 
 describe("USERCONTROL Checkbox Group", () => {
     let checkboxGroup: CheckboxGroupAtom;
     let checkboxGroupJustified: CheckboxGroupAtom;
-    let checkboxGroupInForm: CheckboxGroupAtom;
-    let checkboxGroupInFormWithDisabledItems: CheckboxGroupAtom;
     let checkbox1: CheckboxAtom | undefined;
     let checkbox2: CheckboxAtom | undefined;
     let checkbox3: CheckboxAtom | undefined;
     let checkbox4: CheckboxAtom | undefined;
     let checkboxGroupCheckboxDisabled: CheckboxAtom;
+    let checkboxGroupPartOfForm: CheckboxGroupAtom;
+    let upBtn: ButtonAtom;
 
     beforeAll(async () => {
         checkboxGroup = Atom.find(CheckboxGroupAtom, "nui-demo-checkbox-group-basic");
         checkboxGroupJustified = Atom.find(CheckboxGroupAtom, "nui-demo-checkbox-group-justified");
-        checkboxGroupInForm = Atom.find(CheckboxGroupAtom, "nui-demo-checkbox-group-in-form");
-        checkboxGroupInFormWithDisabledItems = Atom.find(CheckboxGroupAtom, "nui-demo-checkbox-group-in-form-disabled-checkboxes");
-        checkboxGroupCheckboxDisabled = Atom.find(CheckboxAtom , "nui-demo-checkbox-group-1-disabled-checkbox");
+        checkboxGroupCheckboxDisabled = Atom.find(CheckboxAtom, "nui-demo-checkbox-group-1-disabled-checkbox");
+        checkboxGroupPartOfForm = Atom.find(CheckboxGroupAtom, "nui-demo-checkbox-group-part-of-form");
+        upBtn = Atom.findIn(ButtonAtom, element(by.buttonText("Submit")));
     });
 
     beforeEach(async () => {
```

Comment:
It might be nice to see if we can avoid reloading the page before every test like we do on line 25 below. 

```ts
await Helpers.prepareBrowser("checkbox-group/checkbox-group-test");
```

If we can do everything that's currently in the beforeEach in the beforeAll instead, that would speed up this test suite quite a bit probably.

Reply:
If it's too impractical for some reason, don't worry about it.

Reply:
Right, it can help. Moved everything to beforeAll.